### PR TITLE
Add Pandoc convert GitHub Page script

### PR DESCRIPTION
**Pull request by focaaby**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -1,6 +1,9 @@
 all: lkmpg.tex
 	pdflatex -shell-escap lkmpg.tex
 
+html:
+	pandoc -s lkmpg.tex -o lkmpg.html
```
Comment (jserv) [inline Makefile:5]: I checked the HTML file generated by `pandoc`, and It appears that HTML file is not recognizable for source code listing.

Related: https://github.com/sysprog21/lkmpg/pull/15

Comment (focaaby) [reply Makefile:5]: It seems that `pandoc` did not include the `lib/codeblock.tex` properly.

Comment (jserv) [reply Makefile:5]: Since `make4ht` renders HTML better than `pandoc`, I would like to wait for #20 prior to this pull request.

**Comment by jserv** [comment]
Drop pandoc support in favor of #20

**Pull request closed without merging**